pkg/filters/options.go

```diff
@@ -137,10 +140,6 @@ func (o *Options) Namespaces(clientset kubernetes.Interface) []string {
 				return
 			}
 
-			for _, ns := range namespaceList.Items {
```

Comment:
What is this removed for?

Reply:
The removed code (_for loop_)  seems as not needed as it initializes with 'false' all the elements (available namespaces) of namespacesMap and right after that there is a analogical _for loop_ which changes the values to true. The logic of that section of the code is that initialization with 'true' values is the correct one. Both these were added in different commits (PRs) so I guess it was overlooked or there was an idea which was never implemented.